pkg/controller/statefulset/stateful_set.go

```diff
@@ -205,6 +205,20 @@ func (ssc *StatefulSetController) updatePod(old, cur interface{}) {
 		return
 	}
 
+	if curPod.DeletionTimestamp != nil {
```

Comment:
Don't do this.
We've already got other function to handle these to-be-deleted pods.

Reply:
+1 this shouldn't be done here